src/sqlfluff/dialects/dialect_exasol.py

```diff
@@ -773,7 +754,7 @@ class DropCascadeRestrictStatementSegment(BaseSegment):
         ),
         Ref("IfExistsGrammar", optional=True),
         Ref("ObjectReferenceSegment"),
-        OneOf("RESTRICT", "CASCADE", optional=True),
+        Ref("DropBehaviorGrammar", optional=True),
```

Comment:
This `DropCascadeRestrictStatementSegment` should really be split out into the separate drop view and drop function statements to be consistent with other dialects 😄

Reply:
Could you also split out USER and ROLE in the `DropCascadeStatementSegment` too please?

Reply:
also remember to update the `type` of the classes when you do this

Reply:
✔